contractcourt/chain_arbitrator.go

```diff
@@ -397,6 +397,32 @@ func (c *ChainArbitrator) Start() error {
 		}
 
 		c.activeChannels[chanPoint] = channelArb
+
+		// If the channel has had its commitment broadcasted already,
+		// republish it in case it didn't propagate.
+		if !channel.HasChanStatus(channeldb.ChanStatusCommitBroadcasted) {
+			continue
+		}
+
+		closeTx, err := channel.BroadcastedCommitment()
+		switch {
+
+		// This can happen for channels that had their closing tx
+		// published before we started storing it to disk.
+		case err == channeldb.ErrNoCloseTx:
+			log.Warn("Channel %v is in state CommitBroadcasted, "+
+				"but no closing tx to re-publish...", chanPoint)
+			continue
```

Comment:
Any reason we don't craft a new closing transaction in this case?

Reply:
We don't really know which tx was originally broadcasted.

That being said, it wouldn't hurt to attempt re-creating and broadcasting a force close transaction. Would that solve any edge case you think?

Reply:
This is a state that should never really happen. It seems a bit dangerous to attempt to craft a new transaction as we may not be 100% consistent here.